orangecontrib/network/widgets/OWNxAnalysis.py

```diff
@@ -8,6 +8,7 @@
 import Orange
 from Orange.data import Table, Domain
 from Orange.widgets import gui, widget
+from Orange.widgets.settings import *
```

Comment:
We're [against wildcard imports](https://github.com/biolab/orange3/blob/master/CONTRIBUTING.md#coding-style).